admin/includes/init_includes/init_sanitize.php

```diff
@@ -209,7 +209,7 @@
                'group_name', 'geo_zone_name', 'geo_zone_description',
                'tax_class_description', 'tax_class_title', 'tax_description', 'entry_company', 'customers_firstname',
                'customers_lastname', 'entry_street_address', 'entry_suburb', 'entry_city', 'entry_state', 'customers_referral',
-               'symbol_left', 'symbol_right', 'products_model', 'alt_url');
+               'symbol_left', 'symbol_right', 'products_model', 'alt_url', 'email_to', 'email_to_name');

```

Comment:
Is `email_to` supposed to be an email address? If so, it needs to be moved to the group for emails.